src/test/java/org/opensearch/security/dlic/rest/api/TenantInfoActionTest.java

```diff
@@ -12,7 +12,7 @@
 package org.opensearch.security.dlic.rest.api;
 
 import org.apache.hc.core5.http.Header;
-import org.apache.hc.core5.http.HttpStatus;
+import org.apache.http.HttpStatus;
```

Comment:
IMO imports in the `test` package should use `hc.core5`. The issue with automatic backporting is most pronounced in files under the `integrationTest` package.